### PR TITLE
Sara/mkt 631 tier consolidation PART 1

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### In scope:
* Usage Limits FAQ

### NOT in scope
* Removing community tier mentions (because we are not sunsetting community tier yet.)

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
 <iframe title="Semgrep example no prints" src="https://semgrep.dev/embed/editor?snippet=ievans:print-to-logger" width="100%" height="432" frameborder="0"></iframe>
 ```
 
+## Usage limits
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+* All users with any number of contributors are able to access Semgrep Supply Chain, Semgrep Pro Rules, and other paid offerings for free as of this update (May 15, 2023).
```
Comment (dlukeomalley) [inline docs/faq.md:79]: CC @pabloest 

I agree with the concerns from Slack that this implies it's free. I wonder if we may be able to just remove bullets 2 and 3 for the actual FAQ below.

File: docs/faq.md [hunk 2]
```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
...
+* A grace period for all users with any number of contributors is ongoing from now until July 31, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier. This updated Enterprise tier includes such features as single-tenancy and a secure network tunnel, ideal for organizations with increased infrastructure, reporting, and other custom requirements.
```
Comment (dlukeomalley) [inline docs/faq.md:87]: > This updated Enterprise tier includes such features as single-tenancy and a secure network tunnel, ideal for organizations with increased infrastructure, reporting, and other custom requirements.

For now I think we can remove this and just focus on the Community + Team tier, wdyt?

File: docs/faq.md [hunk 3]
```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
...
+
+The 11th contributor and onwards are charged. A 20-member engineering team would pay $7,200 per year for both products at the discounted bundled rate.
+
+For example:
+
+| Product  | List price | Months | Quantity | Subtotal | Discount | Total price |
+| -------  | ------ | ------- | ----- | ------ | ---- | ----- |
+| Semgrep Supply Chain (bundle)  | $30 | 12 | 20 | $7,200 | 10 licenses ($3,600) | $3600 |
+| Code (bundle)  | $30 | 12 | 20 | $7,200 | 10 licenses ($3,600) | $3600 |
+| Total (bundle) |    | | | | | $7200 |
```
Comment (dlukeomalley) [inline docs/faq.md:121]: This table was included in the FAQ more for internal purposes. I think we can remove it from here, and just focus on the text "The 11th contributor and onwards are charged" with perhaps a simple example. What do you think?

File: docs/faq.md [hunk 4]
```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
...
+
+### Can a single product be purchased?
+Yes, you can buy a single product. Products can also be disabled from the Settings page.
+
+### Can I purchase a different number of licenses per product?
+
+If you are purchasing licenses for both products, you can't purchase a different number of licenses per product because the products are bundled together. Both Semgrep Supply Chain and Semgrep Code will run on every scan unless disabled (for all projects) from the settings page.
```
Comment (dlukeomalley) [inline docs/faq.md:128]: @s-santillan, I think the original FAQ was more clear here in that it was more explicit. The answer there was "No" 😅

**Review by dlukeomalley — APPROVED** [review]
Approved, with some asks for minor changes. Thank you!

Would you be open to moving this to its own page? May make it easier for us to link to an early cohort of users for user testing + make it easier to find.

File: docs/faq.md (hunk 2 above)
Comment (s-santillan) [reply docs/faq.md:87]:
```suggestion
```

File: docs/faq.md (hunk 1 above)
Comment (s-santillan) [reply docs/faq.md:79]: Removed this!

File: docs/faq.md (hunk 2 above)
Comment (s-santillan) [reply docs/faq.md:87]: Removed!

File: docs/faq.md (hunk 4 above)
Comment (s-santillan) [reply docs/faq.md:128]: Yeah, have edited this to say "no" with an example.

File: docs/faq.md (hunk 3 above)
Comment (s-santillan) [reply docs/faq.md:121]: Yup, added an example instead.

File: docs/usage-limits.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* By purchasing additional licenses
+* By waiting for the next billing cycle
+
+### What if the first scan exceeds the contributor usage limit?
+
+Semgrep will complete the first scan and a one-time 30-day free trial will automatically start. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### How is the cost calculated for the Team tier?
+
+The 11th contributor and onwards are charged. An engineering team with 20 contributing developers will only pay for 10 licenses.
```
Comment (neo413001) [inline docs/usage-limits.md:50]: The 11th contributor and onwards are charged. For example, an engineering team...

File: docs/usage-limits.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+### Can a single product be purchased?
+
+Yes, you can buy a single product. Products can also be disabled from the Settings page.
```
Comment (neo413001) [inline docs/usage-limits.md:54]: should we hyperlink this? https://semgrep.dev/orgs/-/settings

File: docs/usage-limits.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier.
+
+If the number of contributors has exceeded the usage limit, you will need to purchase licensing for the 11th contributor onward. 
+
+### What is the usage limit?
+
+The usage limit is 10 contributors. A contributor is someone who has made at least one commit to a scanned private repository within the last 30 days.
```
Comment (neo413001) [inline docs/usage-limits.md:30]: ... to a Semgrep scanned private repository...

Comment (s-santillan) [inline docs/usage-limits.md:30]:
```suggestion
The usage limit is 10 contributors. A contributor is someone who has made at least one commit to a Semgrep scanned private repository within the last 30 days.
```

File: docs/usage-limits.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+* By purchasing additional licenses
+* By waiting for the next billing cycle
+
+### What if the first scan exceeds the contributor usage limit?
+
+Semgrep will complete the first scan and a one-time 30-day free trial will automatically start. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### How is the cost calculated for the Team tier?
+
+The 11th contributor and onwards are charged. An engineering team with 20 contributing developers will only pay for 10 licenses.
```
Comment (s-santillan) [inline docs/usage-limits.md:50]:
```suggestion
The 11th contributor and onwards are charged. For example, an engineering team with 20 contributing developers will only pay for 10 licenses.
```

File: docs/usage-limits.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+Semgrep will complete the first scan and a one-time 30-day free trial will automatically start. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### How is the cost calculated for the Team tier?
+
+The 11th contributor and onwards are charged. For example, an engineering team with 20 contributing developers will only pay for 10 licenses.
+
+### Can a single product be purchased?
+
+Yes, you can buy a single product. Products can also be disabled from the Settings page.
```
Comment (s-santillan) [inline docs/usage-limits.md:54]:
```suggestion
Yes, you can buy a single product. Products can also be disabled from the [Settings](https://semgrep.dev/orgs/-/settings) page.
```

**Review by neo413001 — APPROVED** [review]

**Review by neo413001 — APPROVED** [review]

File: docs/usage-limits.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
```
Comment (adamkvitek) [inline docs/usage-limits.md:22]:
```suggestion
The Community tier will sunset on May 23 and Semgrep, Inc offers the Team tier for free to all users up to the usage limit of 10 contributors scanning on private repositories. A contributor is defined as someone who contributed code to a private repository scanned by Semgrep.
```

Comment (adamkvitek) [inline docs/usage-limits.md:22]: @s-santillan Do not use "being scanned" but "scanned". The same applies to "being analyzed" or "being written" - you can use "analyzed" or "written". Apply this to all similar grammatical structures.

Also, avoid using "we" when possible.

File: docs/usage-limits.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier.
+
+If the number of contributors has exceeded the usage limit, you will need to purchase licensing for the 11th contributor onward.
```
Comment (adamkvitek) [inline docs/usage-limits.md:26]:
```suggestion
If the number of contributors exceeds the usage limit, you must purchase licensing for the 11th contributor onward.
```

Comment (adamkvitek) [inline docs/usage-limits.md:26]: Use simple present tense in documentation as much as possible.

Optional: Avoid repetitive overuse of "usage limit". We can use the limit of 10 contributors maybe in some cases?

File: docs/usage-limits.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+Contributors are calculated using `git log` over the past 30 days.
+
+### What happens when the usage limit is exceeded?
+
+Semgrep scans stop when the usage limit is exceeded. You can resume scanning through the following:
+
+* A one-time 30-day free trial (automatically triggered when the usage limit is exceeded for the first time)
+* By purchasing additional licenses
+* By waiting for the next billing cycle
```
Comment (adamkvitek) [inline docs/usage-limits.md:42]: These are all sentences or partial sentences at least... adding full stops at the end. Optional: Remove parenthesis and make the first item one sentence.

```suggestion
* A one-time 30-day free trial that is automatically triggered when the usage limit is exceeded for the first time.
* By purchasing additional licenses.
* By waiting for the next billing cycle.
```

File: docs/usage-limits.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+
+### What if the first scan exceeds the contributor usage limit?
```
Comment (adamkvitek) [inline docs/usage-limits.md:44]:
```suggestion
### What if the first Semgrep scan exceeds the contributor usage limit?
```

File: docs/usage-limits.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+
+Semgrep will complete the first scan and a one-time 30-day free trial will automatically start. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### How is the cost calculated for the Team tier?
+
+The 11th contributor and onwards are charged. For example, an engineering team with 20 contributing developers will only pay for 10 licenses.
```
Comment (adamkvitek) [inline docs/usage-limits.md:50]: Thank you for adding an example!

File: docs/faq.md [hunk 15]
```diff
@@ -186,6 +189,6 @@ A policy is a simple collection of rules and a definition of what to do with rul
 
 ### Do you have a visualization UI?
```
Comment (adamkvitek) [inline docs/faq.md:190]:
```suggestion
### Do you have a graphical user interface?
```

**Review by adamkvitek — COMMENTED** [review]
@s-santillan left a few comments, thank you!

**Review by adamkvitek — APPROVED** [review]

File: docs/faq.md (hunk 15 above)
Comment (adamkvitek) [reply docs/faq.md:190]: @s-santillan I know either user interface (UI) or graphical user interface (GUI). I have not seen a word "visualization UI" before, as I think it comes from "data visualization". See https://stackoverflow.com/questions/3191844/what-is-the-difference-between-gui-and-ui - we should probably use "user interface" looking at the context of this FAQ, what do you think?

```suggestion
### Do you have a user interface?
```

Comment (adamkvitek) [reply docs/faq.md:190]: @s-santillan I was just trying to point out that this is not a standard vocabulary.

**Pull request merged**